std_execution.bs

```diff
@@ -5150,12 +5150,12 @@ are all well-formed.
             in the <code><i>type-list</i></code> named by
             <code>error_types_of_t&lt;copy_cvref_t&lt;S2, S>, E, <i>potentially-throwing</i>></code>
             are `true_type`; otherwise, `completion_signatures<>`, where
-            <code><i>potentially-throwing</i></code> is the template alias:
+            <code><i>potentially-throwing</i></code> is an alias template:
 
             <pre highlight="c++">
             template &lt;class... Es>
-              <i>potentially-throwing</i> =
-                <i>type-list</i>&lt;bool_constant&lt;is_nothrow_invocable_v&lt;F, Es>>...>;
+              using <i>potentially-throwing</i> =
+                <i>type-list</i>&lt;is_nothrow_invocable&lt;F, Es>...>;
```

Comment:
This change is a mistake. The text above specifically says:

> if any of the types in the <code><i>type-list</i></code> named by <code>error_types_of_t&lt;copy_cvref_t&lt;S2, S>, E, <i>potentially-throwing</i>></code> are `true_type`

`is_nothrow_invocable<F, Es>` is never `true_type`.